### PR TITLE
[DDW-488] Fix NumericInput Carrot Malfunction

### PR DESCRIPTION
**Pull request by MarcusHurney**
This PR fixes two bugs in the `NumericInput` component.

1. The first fix addresses the scenario when a user enters numerical digits and then presses `.`. The cursor should move right to the decimal digits, but this intended behavior does not happen when a user enters exactly 4 digits and then presses `.`. It needs an additional press of `.` to move right. 

2. The second fix addresses the scenario when a user enters `ctrl-A` + `del` on an amount being displayed in `NumericInput`, then enters a new amount. The new amount's digits are not in the correct order. For example, the user presses `12345` and gets `23451`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,44 +1,50 @@
 # Changelog
```
Comment (MarcusHurney) [inline CHANGELOG.md]: @DominikGuzei 
This PR is meant to be a hot fix that will be included in the next Daedalus `release/0.12.0`. I just went ahead and assigned the next version as `0.7.2` as it is targeting the master branch. Feel free to change this if needed!

**Comment by nikolaglumac** [comment]
@DominikGuzei can you please take care of releasing this new version on npm so that we can use it in Daedalus? 🙏

**Comment by DominikGuzei** [comment]
@nikolaglumac @MarcusHurney just published `0.7.2` ready for consumption!

**Comment by nikolaglumac** [comment]
Thanks @DominikGuzei - I will bump the version in Daedalus and prepare the PR!

**Comment by nikolaglumac** [comment]
@DominikGuzei @MarcusHurney React-Polymorph version updated in Daedalus: https://github.com/input-output-hk/daedalus/pull/1173
My manual testing shows it works great 🎉 
Thanks for the fix @MarcusHurney 👍

**Pull request merged**